tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/ControlSyntaxSuite.scala

```diff
@@ -0,0 +1,1168 @@
+    )
+  }
+
+  test("old-match-case-inside") {
+    val code = """|x match {
+                  |  case 1 =>
+                  |    y match {
+                  |      case 5 => "5"
+                  |      case 6 => "6"
+                  |    }
+                  |  case 2 =>
+                  |    "2"
+                  |}
+                  |""".stripMargin
+    runTestAssert[Stat](code)(
+      Term.Match(Term.Name("x"), List(Case(Lit.Int(1), None, Term.Match(Term.Name("y"), List(Case(Lit.Int(5), None, Lit.String("5")), Case(Lit.Int(6), None, Lit.String("6"))))), Case(Lit.Int(2), None, Lit.String("2"))))
+    )
+  }
+
+  test("new-match-case-empty") {
+    val code = """|x match
+                  |  case 1 =>
+                  |  case 2 =>
+                  |""".stripMargin
+    val output = """|x match {
+                    |  case 1 =>
+                    |  case 2 =>
+                    |}
+                    |""".stripMargin
+    runTestAssert[Stat](code, assertLayout = Some(output))(
+      Term.Match(
+        Term.Name("x"),
+        List(Case(Lit.Int(1), None, Term.Block(Nil)), Case(Lit.Int(2), None, Term.Block(Nil)))
+      )
+    )
+  }
+
+  test("new-match-case-oneline") {
+    val code = """|x match
+                  |  case 1 => "1"
+                  |  case 2 => "2"
+                  |""".stripMargin
+    val output = """|x match {
+                    |  case 1 => "1"
+                    |  case 2 => "2"
+                    |}
+                    |""".stripMargin
+    runTestAssert[Stat](code, assertLayout = Some(output))(
+      Term.Match(Term.Name("x"), List(Case(Lit.Int(1), None, Lit.String("1")), Case(Lit.Int(2), None, Lit.String("2"))))
+    )
+  }
+
+  test("new-match-case-multiline") {
+    val code = """|x match
+                  |  case 1 =>
+                  |    a1
+                  |    b1
+                  |  case 2 =>
+                  |    a2
+                  |    b2
+                  |""".stripMargin
+    val output = """|x match {
+                    |  case 1 =>
+                    |    a1
+                    |    b1
+                    |  case 2 =>
+                    |    a2
+                    |    b2
+                    |}
+                    |""".stripMargin
+    runTestAssert[Stat](code, assertLayout = Some(output))(
+      Term.Match(Term.Name("x"), List(Case(Lit.Int(1), None, Term.Block(List(Term.Name("a1"), Term.Name("b1")))), Case(Lit.Int(2), None, Term.Block(List(Term.Name("a2"), Term.Name("b2"))))))
+    )
+  }
+
+  // ignored because support is not yet added in parser but unparseable syntax was already identified.
+  // it will be fixed in next batch of changes.
+  test("old-match-case-one-align") {
+    val code = """|cond match {
+                  |  case a =>
+                  |  fa
```

Comment:
Is this fixed? I think we just don't add indentation here. It should be ok

Reply:
all good :+1: